moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -15,7 +15,7 @@ import { formatTimeStampTZType } from "src/utils/dateAndTime";
 export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
   options: {
     useQuery: {
-      fetchPolicy: "cache-first", // Default is "cache-first", or use "no-cache"
+      fetchPolicy: "cache-and-network", // Default is "cache-first", or use "no-cache"
```

Comment:
like i mention in my long comment above—this has the effect of having a project record "jump" to the top of the list view after editing it and then navigating back to the list view. it's not great :/ and we can definitely fix it by being vigilant about refetching in the background. i will open a separate issue for that. i'm curious how urgent y'all think that is...

Reply:
i actually don't mind the jump at all. maybe because it feels like a confirmation that the edit i just made to the project worked? im probably in the minority tho

Reply:
I'm on the fence but lean towards shipping with a focus on fixes coming. I would argue that it is more broken/unoptimized without these changes than with them and having a quirk, but I get that having unexpected things happen in the UI can be annoying and confusing.

Reply:
ok—agreed. we'll ship this and continue to make it better 😅